src/recordlinker/schemas/pii.py

```diff
@@ -133,10 +134,19 @@ class Telecom(pydantic.BaseModel):
 
     model_config = pydantic.ConfigDict(extra="allow")
 
-    value: str  # future use
+    value: str
     system: typing.Optional[str] = None
     use: typing.Optional[str] = None
 
+class DriversLicense(pydantic.BaseModel):
+    """
+    The schema for a Drivers License record
+    """
+
+    model_config = pydantic.ConfigDict(extra="allow")
+
+    value: str
+    authority: str
```

Comment:
@ericbuckley I had the `authority` field as optional but I changed it to required assuming that the authority needs to be present for a valid comparison. Do you agree with that?

Reply:
Yeah that makes sense to me